test/linalg/uniformscaling.jl

```diff
@@ -43,6 +43,15 @@ end
     @test UniformScaling(α)./α == UniformScaling(1.0)
 end
 
+@testset "det and logdet" begin
+    @test isone(det(I))
```

Comment:
Perhaps we should actually change this to `@test det(I) === 1` instead. Since `isone(I)` is true, this will still succeed if `det(::UniformScaling) ` becomes a no-op :)